pkgs/top-level/perl-packages.nix

```diff
@@ -16460,6 +16460,19 @@ let
     };
   };
 
+  TextNSP = buildPerlPackage rec {
```

Comment:
@bzizou You'll need to add this as a separate commit.

Reply:
ok!